Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM ruby:3.0.0
 
-ENV KUBECTL_VERSION=v1.18.2
+ENV KUBECTL_VERSION=v1.20.2
 ENV EJSON_VERSION=1.2.1
-ENV KRANE_VERSION=1.1.3
```

Comment:
Any breaking changes in this bump?

Reply:
Yea, I wanted to ask your opinion about this one already on Friday, but decided that it's not that urgent.

So the only thing mentioned in [release notes](https://github.com/Shopify/krane/blob/v2.0.0/CHANGELOG.md) is removal of [annotation prefix](https://github.com/Shopify/krane/pull/738). I don't think it should matter, but since we pin container version we use in GH action, I was planning to release this as 2.0.0 and then try it out on some sandbox deployment to see what happens.
Looking at resources, at least secrets seem to carry this `kubernetes-deploy.shopify.io` annotation.

Reply:
We haven't set the `kubernetes-deploy.shopify.io` to secrets our selves, but that's added by krane 🤔 I've been pondering about the deprecation warnings about those every now and then. It's good to note that `krane` was called `kubernetes-deploy` before, so I assume this is just cleanup from that migration.

Reply:
Ha, and that automatic annotation was removed in the pr https://github.com/Shopify/krane/pull/738/files#diff-bc3a55f4c9f2690423f2b6ab573b03c92510ebbe3b22719fa124d0f7823f7088L15

Reply:
Yup.